app/templates/community/index.hbs

```diff
@@ -37,86 +34,69 @@
+          even better, submit a pull request.
+        </p>
+        <p>
+          Ember is an inclusive community that welcomes (and benefits from!) people of all backgrounds. If you're having a non-code issue, <a href="mailto:ombudsman@emberjs.com?subject=Website Help Link">reach out</a>. We do our very best to be sure every Ember user has pleasant interactions and positive experiences. If we can help make you feel more
+          comfortable or help navigate a trying situation, let us know.
+        </p>
+      </div>
+      <div class="col-2-large">
+        <img alt="Bug" class="img-fluid" src="/images/community/bug.png">
+      </div>
+    </div>
+  </section>
+  <section class="margin-top-medium" aria-labelledby="section-contribute-to-ember">
+    <div class="layout-grid">
+      <div class="col-2-large">
+        <img alt="Contribute" class="img-fluid" src="/images/community/give.png">
+      </div>
+      <div class="col-4-large">
+        <h2 id="section-contribute-to-ember" class="text-center">In a Giving Mood? Contribute to the Project</h2>
+        <p>
+          The Ember.js source is hosted on <a href="https://github.com/emberjs/ember.js/">Github</a>. To contribute patches, create a fork of the project on GitHub and submit a pull request. Please be sure to include unit tests and documentation for any new features you add. See the <a href="https://github.com/emberjs/ember.js/blob/master/CONTRIBUTING.md">full guidelines</a> for more information.
+        </p>
+      </div>
     </div>
+  </section>
+  <div class="layout-grid">
+    <section class="margin-top-medium col-3-large text-center" aria-labelledby="section-meetups">
+      <h2 id="section-meetups">Feeling Lonely? Join a Meetup</h2>
+      <img alt="Meetup" src="/images/community/meetup.png">
+      <p>
+        Joining an Ember.js meetup is a great way to learn more about Ember, meet other developers, find jobs, etc. Any and all things Ember and local.
+      </p>
+      {{#link-to "community.meetups" class="es-button"}}
+        Find a meetup
+      {{/link-to}}
+    </section>
+
+    <section class="margin-top-medium col-3-large text-center" aria-labelledby="section-mascots">
+      <h2 id="section-mascots">Met The Mascots? You'll Love Them!</h2>
+      <img alt="Campster" src="/images/community/campster.png">
+      <p>
+        Tomster and Zoey have been crafted with love, and are constantly getting makeovers. Events, User Groups, Countries&mdash;sky's the limit!
+      </p>
+      {{#link-to "mascots" class="es-button"}}
+        Meet the Mascots
+      {{/link-to}}
+    </section>
   </div>
-</section>
+  <section class="margin-top-large text-center" aria-labelledby="section-meetup-video">
+    <iframe width="854" height="480" src="https://www.youtube.com/embed/rY5D38RQoEg?rel=0" frameborder="0" allowfullscreen></iframe>
```

Comment:
So the iframe needs `max-width: 100%` on it to make sure that it doesn't bust out of the container on a mobile view 🤔 

I'm personally ok with this being just in a `style=""` attribute but what do other people think?

Reply:
We could throw it inside a grid layout then the 100% max width would always be relative to the wrapper, allowing the grid logic to determine the width on mobile.

Reply:
here is some utility css I have used in the past to set aspect ratio and still allow for dynamic resizing

```
/**
 * Aspect Ratios
**/

/* structure */
.ar-parent {
  position: relative;
  height: 0;
}
.ar-child {
  position: absolute;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  z-index: 100;
  height: 100%;
  width: 100%;
}

/* size options added to the wrapper */
.ratio-square {
  padding-bottom: 100%;
}
.ratio-16x9 {
  padding-bottom: 56.25%;
}
.ratio-9x16 {
  padding-bottom: 177.77%;
}
.ratio-4x3 {
  padding-bottom: 75%;
}
.ratio-3x4 {
  padding-bottom: 133.33%;
}
.ratio-6x4 {
  padding-bottom: 66.6%;
}
.ratio-4x6 {
  padding-bottom: 150%;
}
.ratio-8x5 {
  padding-bottom: 62.5%;
}
.ratio-5x8 {
  padding-bottom: 160%;
}
.ratio-7x5 {
  padding-bottom: 71.42%;
}
.ratio-5x7 {
  padding-bottom: 140%;
}
```

Reply:
here it is locally on mobile 
<img width="716" alt="Screenshot 2019-10-30 17 44 00" src="https://user-images.githubusercontent.com/144861/67904673-124b1c80-fb3d-11e9-9ed8-a95bc9b3f66e.png">

Reply:
Yes I would be happy with using the exiting grid layout here if that solves the issue 🎉 

You might need to make use of the grid helpers that aren't properly documented yet (see https://github.com/ember-learn/ember-styleguide/issues/235) if you want it to be 100% in both desktop and mobile 

`class="col-6-large col-4"`

And thanks for sticking with this implementation, it turns out that it was more complex then I was expecting 🙈

Reply:
Hey @mansona sorry I'm just getting back to this... So to be clear in order to get the screenshot above I had to add the utility css in the [previous comment](https://github.com/ember-learn/ember-website/pull/456#discussion_r340889804). 

I assume that if you are ok with adding that CSS that it needs to go in the Ember Styleguide. I am happy to post a PR for to that repo with the CSS or whatever you think is best. Let me know how to proceed.

Reply:
So we have talked about this at length (twice!) and decided that we're going to add one helper to fix the aspect ratios of videos. You can see more info here: https://github.com/ember-learn/ember-styleguide/issues/244 🎉